app/src/main/java/org/fossasia/openevent/app/event/attendees/AttendeesFragment.java

```diff
@@ -124,6 +152,17 @@ public void onActivityCreated(@Nullable Bundle savedInstanceState) {
 
         fastAdapter.setHasStableIds(true);
         fastAdapter.withEventHook(new AttendeeItemCheckInEvent(this));
+        itemAdapter.withItemFilter(new ItemFilter<>(itemAdapter)
+            .withFilterPredicate(
+                (attendee, query) -> {
+                    int firstNameMatch = FuzzySearch.tokenSortPartialRatio(query.toString().toLowerCase(), attendee.getFirstName().toLowerCase());
+                    if (firstNameMatch > 60) return false;
```

Comment:
token sort partial ratio returns a value in percentage about similarity between two strings. I tested it with some strings and found that  greater than 60 does well. So we are checking similarity of a query with firstname lastname and email of an attendee if at least one of them is greater than 60.

I will add same in the comment